web/src/trigger.tsx

```diff
@@ -6,6 +6,7 @@ export const BuildButtonTooltip = {
   AlreadyQueued: "Resource already queued!",
   NeedsManualTrigger: "Trigger update to sync changes",
   UpdateInProgOrPending: "Resource already updating!",
+  Stop: "Stop this update",
```

Comment:
(grammar nit) i'd prefer `Stop update` to align with `Trigger update` 😅

Reply:
hm, maybe?

I put the "this" to make it less likely to be interpreted as "stop updates for this resource" (like, pause auto trigger), but there's a lot of context to help with that anyway.